Makefile

```diff
@@ -0,0 +1,32 @@
+.PHONY: docker-clean
+docker-clean:
+	# when docker runs as a demon the mounted folders have permissions and cannot be deleted by the current user
+	# so to avoid an annoying "sudo" step another docker container is created to get similar permissions
```

Comment:
Yes, annoying!  I feel this should be fixable in the Dockerfile, I recall seeing another project setting UID/GID of files.  Opened #151

Reply:
Makes sense, should be the way to go